docs/en/modules/adc.md

```diff
@@ -5,9 +5,37 @@
 
 The ADC module provides access to the in-built ADC.
 
-On the ESP8266 there is only a single-channel, which is multiplexed with the battery voltage. Depending on the setting in the "esp init data" (byte 107) one can either use the ADC to read an external voltage, or to read the system voltage, but not both.
+On the ESP8266 there is only a single-channel, which is multiplexed with the battery voltage. Depending on the setting in the "esp init data" (byte 107) one can either use the ADC to read an external voltage, or to read the system voltage (vdd33), but not both.
 
-The default setting in the NodeMCU firmware can be controlled via user_config.h at compile time, by defining one of ESP_INIT_DATA_ENABLE_READVDD33, ESP_INIT_DATA_ENABLE_READADC or ESP_INIT_DATA_FIXED_VDD33_VALUE. To change the setting at a later date, use Espressif's flash download tool to create a new init data block.
+Which mode to use the ADC in can be configured via the `adc.force_init_mode()` function. Note that after switching from one to the other a system restart is required before the change takes effect.
```

Comment:
Actually it's darn obvious - but with a slight twist.
In 0.2.0-rc2 that button with the red icon was labeled 'Reset' but issued `node.restart()`. Note the slight mismatch between "reset" and "restart". In 0.2.0-rc3 the button issues `node.**reset()**` which is now in line with the button label but still a surprise for those used to the old behavior.